smartmoneyconcepts/SMC.py

```diff
@@ -362,6 +436,261 @@ def liquidity(cls, ohlc: DataFrame) -> Series:
         liquidity_end = pd.Series(liquidity_end, name="End")
         liquidity_swept = pd.Series(liquidity_swept, name="Swept")
 
-        return pd.concat(
-            [liquidity, level, liquidity_end, liquidity_swept], axis=1
-        )
\ No newline at end of file
+        return pd.concat([liquidity, level, liquidity_end, liquidity_swept], axis=1)
+
+    @classmethod
+    def volumized_ob(
+        cls,
+        ohlc: pd.DataFrame,
+        maxDistanceToLastBar: int,
+        swingLength: int,
+        obEndMethod: str,
+        maxOrderBlocks: int,
```

Comment:
maxDistanceToBar-it prevents to do not use too big df e.g. if len(df)=3000 it will start looping with i=1750 to i=len(df)
swingLength - yes,it can be the default value = 10 but why should the user not be able to change it?

Reply:
maxDistanceToBar - I believe that if the user wants to limit the amount of data is used then they shouldn't be passing that many candles to the function? Or maybe maxDistanceToBar can be a generic variable that can be used in all functions in the "smc" class
swingLength - yes a default value of 10 is good, and just because you're setting a default doesn't mean the user can't change it, they will still be able to overwrite the value there will just be something there incase they don't write to it.

Reply:
ok